fpdcc/ccfp-asset-dashboard
# add map thumbnail and summary table to project detail page

## Overview
Closes #149

### Demo
![asset-in-project](https://user-images.githubusercontent.com/38969506/168111376-994c8454-183c-4be2-bd56-0128364bb118.gif)


## Testing Instructions
- fetch this branch (still no heroku review apps)
- follow steps in the readme to load the gis and fixture data 
- you'll have to add some assets to a phase
- make sure the map thumbnail and table loads on the project detail page
- also make sure it doesn't load if you're looking at a project that has no assets
